.nuke/build.schema.json

```diff
@@ -126,7 +126,6 @@
               "CheckForbiddenWords",
               "Clean",
               "Compile",
-              "CopySashimiPackagesForConsolidation",
               "CopyToLocalPackages",
               "Pack",
               "PackageConsolidatedCalamariZip",
```

Comment:
This file changes every time we run a local build, which I think adds irrelevant traces to PRs. This file is solely used for [shell completion](https://nuke.build/faq/#build-structure), shall we just gitignore it? It will be auto-generated again locally when the build runs locally next time.

Reply:
I added this file to .gitignore and removed it.